.github/workflows/tests-nightly.yml

```
@@ -471,4 +428,19 @@ jobs:
           JOB_TITLE: "A lot of tests failed, something is probably broken - ${{ needs.run-tests-in-parallel.outputs.dashboard_url }}"
           JOB_KIND: "Tests"
         run: |
-          python3 .github/workflows/notify/notify-slack.py
+          python3 notify/notify-slack.py
```

Doesn't seem valid as it appears that the current working directory is the root of the repo:

```suggestion
          python3 .github/workflows/notify/notify-slack.py
```